### PR TITLE
0.0.2 and make dependency on dependee mandatory.

### PR DESCRIPTION
**Pull request by Vlatombe**
This adds version 0.0.2 for a new test.

### CODE REVIEW

**Comment by ikedam** [comment]
Let me know more details about the context of this change, or I can't review this change.

File: depender/pom.xml [hunk 1]
```diff
@@ -14,10 +14,9 @@
 
   <dependencies>
     <dependency>
-      <groupId>${pom.groupId}</groupId>
+      <groupId>${project.groupId}</groupId>
       <artifactId>dependee</artifactId>
-      <version>${pom.version}</version>
-      <optional>true</optional>
```
Comment (ikedam) [inline depender/pom.xml:20]: Anyway, -1 for this change, as another test in future might need an optional dependency.
It's better to add a new plugin mandatory-depender.
(I should have named this plugin optional-depender!)

**Comment by Vlatombe** [comment]
Hi,

the context is https://github.com/jenkinsci/jenkins/pull/2172 where I'm introducing dependencies version check.

I have added a separate module with an optional dependency so both cases can be tested.

**Comment by ikedam** [comment]
Looks get worse to me.
Optional is an attribute of the depender and introducing a new depender with an mandatory dependency seems simpler and easier to me.

**Comment by Vlatombe** [comment]
You're right, i've modified the patch with your suggestion.

**Comment by Vlatombe** [comment]
By the way, as this module is used for jenkins tests, maybe it would make sense to fork it in the jenkinsci organization?

**Comment by ikedam** [comment]
> By the way, as this module is used for jenkins tests, maybe it would make sense to fork it in the jenkinsci organization?

maybe.

**Pull request merged**